projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/SentenceService.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.api.model.sentence.MainOffence
+import uk.gov.justice.digital.hmpps.api.model.sentence.Offence
+import uk.gov.justice.digital.hmpps.api.model.sentence.SentenceOverview
+import uk.gov.justice.digital.hmpps.integrations.delius.overview.entity.Event
+import uk.gov.justice.digital.hmpps.integrations.delius.sentence.entity.EventSentenceRepository
+import uk.gov.justice.digital.hmpps.integrations.delius.overview.entity.PersonOverviewRepository
+import uk.gov.justice.digital.hmpps.integrations.delius.overview.entity.getPerson
+
+@Service
+class SentenceService(
+    private val personRepository: PersonOverviewRepository,
+    private val eventRepository: EventSentenceRepository
+) {
+    fun getMostRecentActiveEvent(crn: String): SentenceOverview {
+        val person = personRepository.getPerson(crn)
+        val event = eventRepository.findPersonById(person.id)
+            .filter { it.inBreach }
```

Comment:
im not so sure we need to be looking at inbreach here.

Reply:
updated to remove breach check